torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -348,6 +365,14 @@ def set_partitioner_tag(tag: str):
         fx_traceback.current_meta[meta_key] = original_val
 
 
+def set_partitioner_tag_is_backward():
+    return set_partitioner_tag("is_backward")
+
+
+def set_partitioner_tag_must_be_in_backward():
+    return set_partitioner_tag("must_be_in_backward")
```

Comment:
hmm I figured we would just unify thesse two tags into one (since in both cases they're just telling the partitioner "please keep this op in the backward"). any reason to keep them separate?

Reply:
"in_backward" is to mark all backward nodes, which is by default allowed to be moved by partitioner for compute in forward.
"must_be_in_backward" is to prevent this move from backward to forward.

In future we may have more cases, when  we want to keep some particular ops +  "is_backward" condition in backward, but marking them as "must_be_in_backward" will be not handy.